.zenodo.json

```diff
@@ -0,0 +1,103 @@
+{
+  "creators": [
+    {
+      "name": "Tal Yarkoni"
+    },
+    {
+      "affiliation": "University of Texas at Austin",
+      "name": "Alejandro de la Vega",
+      "orcid": "0000-0001-9062-3778"
+    },
+    {
+      "affiliation": "Department of Psychology, Stanford University",
+      "name": "Chris Filo Gorgolewski",
+      "orcid": "0000-0003-3321-7583"
+    },
+    {
+      "affiliation": "Stanford University",
+      "name": "Christopher J. Markiewicz",
+      "orcid": "0000-0002-6533-164X"
+    },
+    {
+      "affiliation": "Florida International University",
+      "name": "Taylor Salo",
+      "orcid": "0000-0001-9813-3167"
+    },
+    {
+      "name": "Krista DeStasio"
+    },
+    {
+      "affiliation": "Dartmouth College: Hanover, NH, United States",
+      "name": "Yaroslav Halchenko",
```

Comment:
```suggestion
      "name": "Yaroslav O. Halchenko",
```

Reply:
Can you update yourself in the `.mailmap`, as well, to ensure that this name matches the `shortlog` entry?